src/main/java/com/hotels/hcommon/hive/metastore/compatibility/HiveMetaStoreClientCompatibility12x.java

```diff
@@ -61,7 +61,7 @@ public HiveMetaStoreClientCompatibility12x(IMetaStoreClient client) {
         field.setAccessible(false);
       }
       return result;
-    } catch (Exception e) {
+    } catch (SecurityException | NoSuchFieldException | IllegalArgumentException | IllegalAccessException e) {
```

Comment:
More specific errors caught because of [REC_CATCH_EXCEPTION](http://findbugs.sourceforge.net/bugDescriptions.html#REC_CATCH_EXCEPTION)